.gitignore

```diff
@@ -13,6 +13,9 @@ crash.log
 # macOS artifacts
 .DS_Store
 
+# IntelliJ artifacts
+.idea
```

Comment:
We don't add IDE-specific files to our .gitignores, but probably this unrelated change just slipped in by accident. Could you remove it?

Reply:
yes I overlooked this change. It's reverted